slicing_copy.lua

```diff
@@ -104,19 +108,19 @@ local function cut(shift, endpos)
         :arg("-v", "warning")
         :arg(o.overwrite and "-y" or "-n")
         :arg("-stats")
-    if ua and ua ~= '' and ua ~= 'libmpv' then
-        cmds:arg('-user_agent', ua)
+    if is_remote() and ua and ua ~= '' and ua ~= 'libmpv' then
+        cmds:arg('-user_agent', "\""..ua.."\"")
     end
     if referer and referer ~= '' then
         cmds:arg('-referer', referer)
     end
     cmds:arg("-ss", tostring(shift))
-        :arg("-i", inpath)
-        :arg("-t", tostring(endpos - shift))
-        :arg("-c:v", o.vcodec)
-        :arg("-c:a", o.acodec)
-        :arg(not copy_audio and "-an" or nil)
-        :arg(outpath)
+    cmds:arg("-i", inpath)
+    cmds:arg("-t", tostring(endpos - shift))
+    cmds:arg("-c:v", o.vcodec)
+    cmds:arg("-c:a", o.acodec)
+    cmds:arg(not copy_audio and "-an" or nil)
+    cmds:arg(outpath)
```

Comment:
Why add these `cmds`? Is this a convention in lua community?

Reply:
I have no idea about this, I did so because in the process I changed the order of the arguments, you can leave or delete, I do not think it really important